tests/credentials.json

```diff
@@ -4,12 +4,5 @@
      "HUB": "IBMQ_HUB", 
      "GROUP": "IBMQ_GROUP", 
      "PROJECT": "IBMQ_PROJECT"
- }, 
```

Comment:
Why was this removed? Don't we need it for GitHub testing?

Reply:
Is it because we are reading the credentials already configured on the machine we are running the tests (i.e. `q128`)?